src/applications/personalization/profile-2/components/ProfileInfoTable.jsx

```diff
@@ -1,37 +1,109 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const ProfileInfoTable = ({ data, dataTransformer, fieldName, title }) => (
-  <table className="profile-info-table" data-field-name={fieldName}>
```

Comment:
I switched from rendering a `table` to building the "table" with `div`s instead. The main reason for this was because the layout of the table needed to change on mobile. On larger screens the table has two columns. On smaller screens all the content is shows in a single column. Achieving that with a real `table` meant rendering one of two different table layouts, depending on the screen width. Switching to `div`s and using flexbox allows for writing the DOM once.